backend/flask_api/map.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask, jsonify, request
+import json
+
+app = Flask(__name__)
+
+@app.route('/', methods=['GET'])
+def home():
+    info = "./map<br>" \
+           "./sector"
+    return info
+
+@app.route('/map', methods=['GET', 'POST'])
+def map_view():
+    directory_path = '../../utils/json_maps/filled_100x100.json' # TODO probably changed when we create some databases
+    with open(directory_path) as file:
+        return json.load(file)
+    # name = request.args.get('forest_name')
+
+@app.route('/sector', methods=['GET', 'POST'])
+def sector_view():
+    directory_path = '../../utils/json_maps/filled_100x100.json'  # TODO probably changed when we create some databases
+    with open(directory_path) as file:
+        name = request.args.get('sector_name')
+        sectors = json.load(file)["sectors"]
+        for sector in sectors:
+            if sector['sector_name'] == name:
+                return sector
+    # name = request.args.get('forest_name')
+    # sector_name = request.args.get('forest_name')
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Comment:
Github podpowiada że brakuje newline na końcu pliku, wiem że to pedantyzm ale na IO twierdzili że nawet takie rzeczy powinno się pisać.